.circleci/config.yml

```diff
@@ -92,8 +92,10 @@ jobs:
       - run: |
           # NOTE: -Werror disabled in CI, as we currently have >4k warnings.
           cd build \
+            && apt-get update && apt-get install --assume-yes lld \
```

Comment:
should add this to the docker build container dependencies script

Reply:
Yup, I added this under the commit marked as revert. For now wanted to test it in ci and get PR feedback before tweaking the dockerfile (which ideally is in a separate PR merged first).

Flow could be:
 1) Merge second PR that adds `lld` to dockerfile
 2) That triggers an update of the base-image on merge to master
 3) Use updated baseimage with `lld` in this PR.

Reply:
Once https://github.com/valhalla/valhalla/pull/2819 is merged I can rebase and remove this `apt-get install`